addons/base_setup/views/base_config_settings_view.xml

```diff
@@ -1,27 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data>
+<odoo>
         <record id="view_general_configuration" model="ir.ui.view">
-            <field name="name">General Settings</field>
+            <field name="name">base.config.settings.view.form</field>
             <field name="model">base.config.settings</field>
             <field name="arch" type="xml">
                 <form string="General Settings" class="oe_form_configuration">
                     <header>
-                        <button string="Apply" type="object" name="execute" class="oe_highlight"/>
-                        <button string="Cancel" type="object" name="cancel" class="oe_link"/>
+                        <button name="execute" string="Apply" type="object" class="oe_highlight"/>
+                        <button name="cancel" string="Cancel" type="object" class="oe_link"/>
```

Comment:
Is it really need difference ?
![gifd](https://cloud.githubusercontent.com/assets/14940179/12124185/e66e15cc-b3a1-11e5-977e-af5e44082944.gif)